test/factories/assignments.rb

```diff
@@ -2,7 +2,9 @@
   factory :assignment do
     client
     volunteer
-    state 'suggested'
+    state ['suggested', 'active'].sample
     association :creator, factory: :user
+    period_start Faker::Date.between(500.days.ago, 200.days.ago)
+    period_end [nil, Faker::Date.between(199.days.ago, 10.days.ago)].sample
```

Comment:
We should have a test for all the factories, there's `FactoryGirl.lint` now which makes this very easy. Not sure why [they recommend putting it into a Rake task though](https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#linting-factories), I would just try adding a test which calls `FactoryGirl.lint traits:true`.

Reply:
I'm not sure if that is in the scope of my story...

Reply:
... and as there is quite some other things touched, I can write an issue or a story for this, but don't see FactoryTesting as my scope now. sorry...

Reply:
Fair enough, I'll add this myself :)